### PR TITLE
French

### PR DESCRIPTION
**Pull request by el00ruobuob**
This is the French Localization of Monerujo.

It includes the new folder along with the three files fully translated.

I may have missed a few typos as usual, and it may includes incoherencies, especially regarding the infinitive / conjugated forms of verbs in the strings that were sometimes hard to guess from the string name or existing translations.
So i believe a few corrections should be made after seeing this "live" on my smartphone.

@erciccione i let you ping our french speakers?

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,375 @@
+<resources>
+    <string name="app_name" translatable="false">monerujo</string>
+    <string name="login_activity_name" translatable="false">monerujo</string>
```
Comment (m2049r) [inline app/src/main/res/values-fr/strings.xml]: please delete all strings with ```translatable="false"``` - they may only be in the original files.

**Review by m2049r — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (m2049r) [reply app/src/main/res/values-fr/strings.xml]: also, i think you may be working off an old version. please rebase to current master.

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: ok, done.

**Comment by m2049r** [comment]
@el00ruobuob sorry - i meant those strings should not be translated and completely removed from the translations.

**Comment by el00ruobuob** [comment]
Remove like this?
It's also pulled and rebase on latest.

**Comment by m2049r** [comment]
> Remove like this?

perfect, thanks!

the following strings from ```strings.xml``` are not translated (probably because they were not in the master branch when you pulled it first):
```XML
    <string name="menu_changepw">Change Passphrase</string>
    <string name="changepw_progress">Change Password in progress</string>
    <string name="changepw_failed">Change Password failed!</string>
    <string name="changepw_success">Password changed</string>
    <string name="prompt_changepw">New Passphrase for %1$s</string>
    <string name="prompt_changepwB">Repeat Passphrase for %1$s</string>
    <string name="generate_bad_passwordB">Passphrases do not match</string>
    <string name="generate_empty_passwordB">Passphrase may not be empty</string>
    <string name="generate_crazypass_label">Wallet Files Restore Password</string>
```

also there is a new a chapter "Wallet Files Recovery Password" in ```help_details``` of ```help.xml``` which is new.

**Comment by el00ruobuob** [comment]
Ok, i'm doing them.
Btw, i wonder how to translate CrAzYpass, or if the term should be translated at all.

**Comment by el00ruobuob** [comment]
@m2049r Enjoy!

**Comment by erciccione** [comment]
Pinging @assylias @glv2 @cryptochangements34

**Comment by m2049r** [comment]
@el00ruobuob merci beaucoup!

yes, CrAzYpass is not to be translated :)

**Comment by m2049r** [comment]
heh - please remove:

- ```info_send_xmrto_success_status_label```
- ```info_send_xmrto_success_status```

from ```strings.xml```. Also ```'``` need to be escaped as ```\'```.

File: app/src/main/res/values-fr/help.xml [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+        <h1>Détails du Portefeuille</h1>
+        <h2>Adresse Publique</h2>
+        Votre adresse publique est comme votre numéro de compte en banque, vous pouvez la partager avec n’importe qui sans
+        craindre de perdre vos Moneroj. On porra envoyer des Moneroj dans votre portefeuille en utilisant cette adresse.
+        <h2>Phrase Mnémonique</h2>
+        C’est la seule donée nécessaire pour récupérer votre portefeuille ultérieurement et obtenir l’accès à
+        vos fonds. Il est très imprtant de la garder sécurisée et confidentielle, car elle donne <em>à quiconque</em>
+        le contrôle total de vos Moneroj ! Si vous ne l’avez pas écrite à un emplacement sécurisé, faites-le !
+        <h2>Mdp de restauration des fichiers du Portefeuille</h2>
+        Assurez vous de noter ce mot de passe. Si vous réinitialisez votre appareil ou désinstallez l'application,
```
Comment (m2049r) [inline app/src/main/res/values-fr/help.xml]: pls excape the ```'``` - why is the apostrophe a different one than e.g. "n’importe" or "l’accès"?

Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: Cause on french keyboard we use to take a single quote as an apostrophe... I just corected that, removed the to xmrto strings mention above, and escaped everything in strings.xml

File: app/src/main/res/values-fr/help.xml [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+        Le mot de passe permet de sécuriser les données de votre portefeuille sur votre appareil.
+        Utilisez un mot de passe fort - mieux, utilisez une phrase secrète.<p>
+        <p>Entrez votre adresse Monero dans le champ \"Adresse Publique\" et remplissez \"Clef d’Audit\" et \"Clef de dépense\".</p>
+        <p>Si vous connaissez le numéro du bloc de votre première transaction à cette adresse, entrez-le dans le
+        champ \"Hauteur de restauration\" - en le laissant vide la <em>totalité</em> de la chaîne de blocs sera scannée
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Créer un Portefeuille - Audit</h1>
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/help.xml]: maybe make this "Portefeuille d'Audit" to match the GUI ?

File: app/src/main/res/values-fr/help.xml [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+        <h3>Archivage</h3>
+        <p>Faites une sauvegarde du portefeuille suivi de sa suppression. La copie reste dans le répertoire
+        <tt>backups</tt>. Si vous n’avez plus besoin de vos sauvegardes, vous devriez les supprimer avec
+        une application d’explorateur de fichiers ou une application de suppression sécurisée.</p>
+    ]]></string>
+
+    <string name="help_wallet"><![CDATA[
+        <h1>Le Portefeuille</h1>
+        <h2>Analyse</h2>
+        Comme Monero aime garder les choses confidentielles, nous devoins scanner la chaîne de blocs
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/help.xml]: did you mean "nous devons" ?

File: app/src/main/res/values-fr/help.xml (hunk 3 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: sure.

File: app/src/main/res/values-fr/help.xml (hunk 4 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: i think so.

File: app/src/main/res/values-fr/help.xml [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        un temps limité - vous pouvez remarquer le compte à rebours sur le bouton \"Dépenser\". Ce taux
+        de change peut être différent du taux indicatif de l’écran précédent.</p>
+        <h2>Clef Secrète XMR.TO<h2>
+        <p>Comme Monerujo ne traite que le volet Monero de votre transaction, votre clef secrète
+        XMR.TO peut être utilisée pour suivre le volet Bitcoin de votre ordre sur la page d’accueil
+        de XMR.TO.</p>
+        <p>Veuillez noter que cette clef secrète est seulement valable 24h après le lancement de la
+        transaction !</p>
+        <h2>Compte à Rebours XMR.TO !</h2>
+        <p>Une fois qu le compte à rebours atteint zéro, vous devrez obtenir un nouveau devis depuis
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/help.xml]: Une fois **que**

File: app/src/main/res/values-fr/help.xml [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/help.xml]: une fois *que*

File: app/src/main/res/values-fr/strings.xml [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
+    <string name="rename_failed">Renommage Échoué !</string>
+    <string name="changepw_failed">Modification du mot de passe Échouée !</string>
+    <string name="changepw_success">Modification du mot de passe réussie</string>
+
+    <string name="label_daemon">Nœud</string>
+    <string name="prompt_daemon">([&lt;utilisateur&gt;:&lt;mdp&gt;@]&lt;démon&gt;[:&lt;port&gt;])</string>
+    <string name="prompt_mainnet">Sélection Réseau</string>
+    <string name="connect_stagenet">StageNet</string>
+    <string name="connect_testnet">TestNet</string>
+    <string name="connect_mainnet">MainNet</string>
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/strings.xml]: shouldn't it just be "Stagenet", "Testnet", and "Mainnet"? That's the way it is done in the official CLI/GUI

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: I kept this unchanged. global change @m2049r ?

File: app/src/main/res/values-fr/help.xml (hunk 6 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: damned!

File: app/src/main/res/values-fr/strings.xml (hunk 7 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: I can lowercase the net part in every language if you're ok.

File: app/src/main/res/values-fr/help.xml (hunk 5 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: done

File: app/src/main/res/values-fr/strings.xml [hunk 8]
```diff
@@ -0,0 +1,366 @@
...
+    <string name="label_send_progress_create_tx">Préparation de la Transaction Monero</string>
+
+    <string name="label_send_progress_queryparms">Interrogation des paramètres xmr.to</string>
+
+    <string name="label_generic_xmrto_error">ERREUR XMR.TO</string>
+    <string name="text_generic_xmrto_error">Code : %1$d</string>
+
+    <string name="text_retry">Touchez pour réessayer</string>
+    <string name="text_noretry_monero">Maintenant on est coincé ici !</string>
+    <string name="text_noretry">Oh-oh, XMR.TO n\'a pas laire disponible pour le moment !</string>
```
Comment (cryptochangements34) [inline app/src/main/res/values-fr/strings.xml]: " l'air " not " laire "

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: done. don't know where it came from...

File: app/src/main/res/values-fr/strings.xml (hunk 7 above)
Comment (m2049r) [reply app/src/main/res/values-fr/strings.xml]: i'm very ok with that . thanks @el00ruobuob

Comment (m2049r) [reply app/src/main/res/values-fr/strings.xml]: hmm - unless they are the same in every language, in which case we should mark them untranslatable and remove them from the translations and fix only the original?

Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: IMO they should be the same in all languages. I'll do the change in values/strings.xml, add the untranslatable to it, and remove the three strings from other strings.xml. (after a rebase for norvegian)

**Review by m2049r — APPROVED** [review]
thank you for your patience!

**Comment by m2049r** [comment]
@cryptochangements34 let me know if it's fine with you too

**Comment by erciccione** [comment]
Hey @el00ruobuob we got some new strings after #244 got merged. Can you rebase and translate the new strings as well?

**Comment by el00ruobuob** [comment]
@erciccione @m2049r rebased it is.
I did translate those new strings from #244 & #254 also.

**Pull request merged**